src/components/tiger/tiger.tsx

```diff
@@ -0,0 +1,252 @@
+import { useState, useEffect, useCallback } from "react";
+import { Button, Stack, TextInput, Checkbox } from "@mantine/core";
+import { useForm } from "@mantine/form";
+import { CommandHelper } from "../../utils/CommandHelper";
+import ConsoleWrapper from "../ConsoleWrapper/ConsoleWrapper";
+import { SaveOutputToTextFile_v2 } from "../SaveOutputToFile/SaveOutputToTextFile";
+import { RenderComponent } from "../UserGuide/UserGuide";
+import InstallationModal from "../InstallationModal/InstallationModal";
+import { LoadingOverlayAndCancelButton } from "../OverlayAndCancelButton/OverlayAndCancelButton";
+import { checkAllCommandsAvailability } from "../../utils/CommandAvailability";
+
+/**
+ * Represents the form values for the Tiger component.
+ */
+interface FormValuesType {
+    targetIP: string;
+    auditLevel: string;
+    reportFile: string;
+    enableModules: string;
+    excludeModules: string;
+}
+
+/**
+ * The Tiger component.
+ * @returns The Tiger component.
+ */
+const Tiger = () => {
+    // Component state variables
+    const [loading, setLoading] = useState(false); // State variable to indicate loading state
+    const [output, setOutput] = useState(""); // State variable to store the output of the command execution
+    const [allowSave, setAllowSave] = useState(false); // State variable to allow saving of output
+    const [hasSaved, setHasSaved] = useState(false); // State variable to indicate if output has been saved
+    const [isCommandAvailable, setIsCommandAvailable] = useState(false); // State variable to check if the command is available.
+    const [opened, setOpened] = useState(!isCommandAvailable); // State variable that indicates if the modal is opened.
+    const [loadingModal, setLoadingModal] = useState(true); // State variable to indicate loading state of the modal.
+    const [pid, setPid] = useState(""); // State variable to store the process ID of the command execution.
+    const [verboseMode, setVerboseMode] = useState(false); // State variable for verbose mode
+
+    // Component Constants
+    const title = "Tiger";
+    const description =
+        "Tiger is a security audit and intrusion detection tool designed to audit Unix-based systems for security issues.";
+    const steps =
+        "=== Required ===\n" +
+        "Step 1: Input a target IP or hostname to audit.\n" +
+        "Step 2: Input the desired audit level (e.g., 1 for basic, 5 for deep).\n" +
+        " \n" +
+        "=== Optional ===\n" +
+        "Step 3: Specify a file to save the audit report.\n" +
+        "Step 4: Enable specific modules for the audit by entering module names.\n" +
+        "Step 5: Exclude specific modules from the audit by entering module names.\n" +
+        "Step 6: Check the verbose mode box for detailed output.\n";
+    const sourceLink = ""; // Link to the source code
```

Comment:
Need to include the source link. 
```suggestion
    const sourceLink = "https://www.kali.org/tools/tiger/"; // Link to the source code
```